### PR TITLE
Upgrade and Fix: React Native Maps (0.71.7)

### PR DESCRIPTION
**Pull request by cbshoaib**
#  Updated-Readme: React Native Maps

## Type of PR

- [ ] Bugfix
- [ ] New feature
- [x] Minor changes

## Changes introduced
added module dependencies installation setup and scope features in README.md file.

### CODE REVIEW

**Review by alaisgomes — APPROVED** [review]

File: modules/react-native-maps/README.md [hunk 1]
```diff
@@ -1,6 +1,27 @@
...
 
+## Scope Features
+The following are the key features in scope for this module. 
+
+1. Ability to display the map
+2. Ability to set the custom markers
+3. Ability to add custom annotations on the map
+
+## Install Required Dependencies/Packages
+All the required packages are given in the `package.json` file. Make sure all the dependencies are installed before using this module. Copy all the packages from the `dependencies` and `x-dependencies` section and past them in your project's main `package.json` file.
```
Comment (danielsousaio) [inline modules/react-native-maps/README.md:12]: > Copy all the packages from the `dependencies` and `x-dependencies` section and past them in your project's main `package.json` file.

Our platform already does this, users don't have to do it manually.

File: modules/react-native-maps/README.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
+Here are the required packages for the module:
+```
+"react-native-maps": "0.27.1",
+"react-native-maps-directions": "1.8.0",
+"react-native-google-places-autocomplete": "2.4.1",
+"prop-types": "15.8.1"
```
Comment (danielsousaio) [inline modules/react-native-maps/README.md:18]: Same here, our platform already installs dependencies.

**Review by danielsousaio — CHANGES_REQUESTED** [review]
The install section can be removed. More details below.

**Review by saad-abid-crowdbotics — APPROVED** [review]
LGTM.

**Review by danielsousaio — APPROVED** [review]

**Pull request merged**